include/ginkgo/core/multigrid/pgm.hpp

```diff
@@ -189,6 +190,11 @@ class Pgm : public EnableLinOp<Pgm<ValueType, IndexType>>,
 
     void generate();
 
+    std::tuple<std::shared_ptr<LinOp>, std::shared_ptr<LinOp>,
```

Comment:
Eventhough it is not a public method, some short documentation on what generate_local is expected to do, would be good.